### DISCUSSION TITLE
How do I manually trigger the document generation?

### DISCUSSION THREAD

**Discussion by jordanpadams** [pds-data-dictionaries/PDS-Data-Dictionaries.github.io#22] [Q&A] [answered]
I updated my docs on branch and the docs didn't generate as I would expect. Can I manually trigger the automated doc generation?

**Comment by pds-ops** [comment, marked as answer]
Yes. A new feature just added to all the repos is the ability to manually trigger the doc generation process:

❗**WARNING**❗ Do not trigger this and the LDD generation process at the same time. Their commits will collide and it will be a headache.

1. Go to the `Actions` tab in your repo
2. From the left panel under `All Workflows`, select `Build and Deploy Docs`. Or replace `ldd-template` with your repo from this URL: https://github.com/pds-data-dictionaries/ldd-template/actions/workflows/build-docs.yml
3. Select the `Run Workflow` dropdown, select the branch you would like to build the docs on, and `Run Workflow`.
<img width="1087" alt="Screenshot 2023-01-23 at 10 22 24 AM" src="https://user-images.githubusercontent.com/93455202/214119132-9c6bd50a-a144-4b12-b870-5e9f497a6a18.png">
